lib/googleGroups.ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest } from "next"
+import allowedGroups from "../config/allowedGroups"
+import cookie from "cookie"
+import jwt from "jsonwebtoken"
+
+/** check if the logged in user is in any of the groups allowed to log in. ONLY WORKS ON A HACKNEY DOMAIN */
+export const checkAuthorisedToLogin = async (
+  req: NextApiRequest
+): Promise<boolean> => {
+  // if we're in local dev, allow any user
+  if (process.env.NODE_ENV === "development") return true
```

Comment:
Just wondering how this interacts with test environment? My guess is it's fine but just prompting to see if this is an issue.

Reply:
Discussed offline, we’re in a test environment when we use Cypress and in our tests we log in by visiting as a user with a session token which is seeded into the database before the tests are run.